teams-fluid-hello-world/src/client/helloWorldTab/FluidContent.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { Button } from "@fluentui/react-northstar";
+import { SharedMap } from "fluid-framework";
+import { diceValueKey } from "./Util";
+import { useState, useEffect } from "react";
+
+export type FluidContentProps = Readonly<{
+    fluidMap: SharedMap;
+}>
+
+export const FluidContent = (props : FluidContentProps) => {
+
+    const [diceView, setDiceView] = useState<{color : string, content : string} | undefined >(undefined);
```

Comment:
Since you're passing in the SharedMap here you can get strong typing on this object.

```suggestion
    const [diceView, setDiceView] = useState<{color : string, content : string}>(props.fluidMap.get(diceValueKey));
```